data/858/659/57/85865957.geojson

```diff
@@ -26,12 +26,10 @@
     "mz:tier_locality":7,
     "mz:tier_metro":1,
     "name:eng_x_preferred":[
-        "Panassus"
+        "Parnassus"
     ],
     "name:eng_x_variant":[
-        "Panassus Ashbury Heights"
-    ],
-    "name:und_x_variant":[
+        "Parnassus Ashbury Heights",
         "Parnassus Heights"
```

Comment:
Do we need to add the misspelling to the variants list to comply with our lifecycle guidelines and not churn the ID?

Reply:
Added the "Panassus" name back in.. we've been lenient when it comes to editing misspellings in place, but I think it's fine to add the variant in this case.